dev-aws/kafka-shared/iam.tf

```diff
@@ -127,6 +127,7 @@ module "iam_jwks_publisher" {
 module "iam_identitydb_event_forwarder" {
   source           = "../../modules/tls-app"
   produce_topics   = [kafka_topic.iam_identitydb_v1.name]
+  consume_topics   = { (kafka_topic.iam_revoked_v1.name) : "iam-identitydb-event-forwarder" }
```

Comment:
For consumer groups can we use the same "iam." convention, please. So this would be "iam.identitydb-event-forwarder"

Reply:
oh fair enough, sorry it's a bit inconsistent so didn't realise but will update now 👍

Reply:
Yea, you're right. I should write those down in a README too.